data/cards-expansion_set-shatteredAlliances.cfg

```diff
@@ -163,4 +180,74 @@
 		
 		}
 	},
+	
+	"I am Legion": {
+		name: "I am Legion",
+		rarity: 2,
+		set: "shatteredAlliances",
+		type: "creature",
+		school: "@eval BLOOD",
+		flavor_text: '"My name is Legion", he replied, "...for we are many."', //how to make parts of it italics?  
+		portrait: "flesh-golem.png",
+		portrait_focus: [340,222],
+		portrait_focus_height: 200,
+		artist: "wendyyoon",
+		cost: 3,
+		loyalty_cost: 3,
+		
+		creature: {
+			game_sounds: "ghostly_creature",
+			sound_summoned: "creatures/summon-flesh-golem.wav",
+			sound_death: "creatures/death-flesh-golem.wav",
+			tags: ['Construct', 'Golem'],
+			attack: 0,
+			life: 0,
+			triggered_abilities: [{
+				name: "Horrific Meld",
+				rules: "Has <font tag='attack'>X</font>attack/<font tag='life'>Y</font>life - on summoned <font tag='attack'>X</font> and <font tag='life'>Y</font> is set to the total attack and total life of all your other creatures. Obliterate every other creature you control.",
+				on_summoned: "def(class creature creature, class game_state game) ->commands
+				[
+					[
+						[
+						set(c.destroyed, true),			
+						set(c.obliterated, true)
+						] | c <- filter(game.creatures, not value.is_building and (value.controller = creature.controller)), c != creature
+					],
+					[
+						[
+						set(creature.life, summ_life)  where summ_life = sum(map(my_creatures,value.life)),			
+						set(creature.attack, summ_attack) where summ_attack = sum(map(my_creatures,value.attack))
+						] where my_creatures = filter(game.creatures, not value.is_building and (value.controller = creature.controller))
+					]	
+				]",
+			}],
+		}
+	},
+	
+//Aether cards
+	
+	"Quantum in oculis deorum": {
+		name: "Quantum in oculis deorum",
+		rarity: 2,
+		set: "shatteredAlliances",
+		type: "spell",
+		school: "@eval AETHER",
+		flavor_text: "All are equal in the eyes of gods",
+		portrait: "sul-dahl-devout.png",
+		cost: 3,
+		loyalty_cost: 3,
+		rules: "All creatures life is equilized - rounded down. Same with attack but rounded up.",
+		
+		on_play: "def(class game_state game, class message.play_card info) ->commands [
+			map(creature_list[0:], set(value.life, new_life) where new_life =  sum(map(creature_list,value.life))/size(creature_list))
+				where creature_list = filter(game.creatures, (value.is_building = false) and (value.is_land = false)),
```

Comment:
Related to: https://github.com/davewx7/citadel/blob/57344282748ecbc42e41c6288915fdaa834dff64/data/classes/card_base.cfg#L42-L44